fast64_internal/oot/oot_cutscene.py

```diff
@@ -0,0 +1,604 @@
+		attrName = "bgm"
+	elif listProp.listType == "Misc":
+		attrName = "misc"
+	elif listProp.listType == "0x09":
+		attrName = "nine"
+	elif listProp.listType == "Unk":
+		prop_split(box, listProp, 'unkType', 'Unk List Type')
+		attrName = "unk"
+	else:
+		raise PluginError("Invalid listType")
+		
+	dat = getattr(listProp, attrName)
+	for i, p in enumerate(dat):
+		p.draw(box, listProp, listIndex, i, objName, collectionType)
+	if len(dat) == 0:
+		box.label(text = "No items in " + listProp.listType + " List.")
+	if listProp.listType == "Textbox":
+		row = box.row(align=True)
+		for l in range(3):
+			addOp = row.operator(OOTCSTextboxAdd.bl_idname, text = 'Add ' + ootEnumCSTextboxType[l][1], icon = ootEnumCSTextboxTypeIcons[l])
+			addOp.collectionType = collectionType + '.textbox'
+			addOp.textboxType = ootEnumCSTextboxType[l][0]
+			addOp.listIndex = listIndex
+			addOp.objName = objName
+	else:
+		addOp = box.operator(OOTCollectionAdd.bl_idname, text = 'Add item to ' + listProp.listType + ' List')
+		addOp.option = len(dat)
+		addOp.collectionType = collectionType + '.' + attrName
+		addOp.subIndex = listIndex
+		addOp.objName = objName
+
+
+class OOTCSListAdd(bpy.types.Operator):
+	bl_idname = 'object.oot_cslist_add'
+	bl_label = 'Add CS List'
+	bl_options = {'REGISTER', 'UNDO'} 
+
+	collectionType : bpy.props.StringProperty()
+	listType : bpy.props.EnumProperty(items = ootEnumCSListType)
+	objName : bpy.props.StringProperty()
+
+	def execute(self, context):
+		collection = getCollection(self.objName, self.collectionType, None)
+		collection.add()
+		collection[len(collection)-1].listType = self.listType
+		#self.report({'INFO'}, 'Success!')
+		return {'FINISHED'} 
+
+def drawCSAddButtons(layout, objName, collectionType):
+	def addButton(row):
+		nonlocal l
+		op = row.operator(OOTCSListAdd.bl_idname, text = ootEnumCSListType[l][1], icon = ootEnumCSListTypeIcons[l])
+		op.collectionType = collectionType
+		op.listType = ootEnumCSListType[l][0]
+		op.objName = objName
+		l += 1
+	box = layout.column(align=True)
+	l = 0
+	row = box.row(align=True)
+	row.label(text = 'Add:')
+	addButton(row)
+	for _ in range(3):
+		row = box.row(align=True)
+		for _ in range(3):
+			addButton(row)
+	box.label(text = 'Install zcamedit for camera/actor motion.')
+
+
+class OOTCutsceneProperty(bpy.types.PropertyGroup):
+	csEndFrame : bpy.props.IntProperty(name = "End Frame", min = 0, default = 100)
+	csWriteTerminator : bpy.props.BoolProperty(name = "Write Terminator (Code Execution)")
+	csTermIdx : bpy.props.IntProperty(name = "Index", min = 0)
+	csTermStart : bpy.props.IntProperty(name = "Start Frm", min = 0, default = 99)
+	csTermEnd : bpy.props.IntProperty(name = "End Frm", min = 0, default = 100)
+	csLists : bpy.props.CollectionProperty(type = OOTCSListProperty, name = 'Cutscene Lists')
+
+def drawCutsceneProperty(box, obj):
+	prop = obj.ootCutsceneProperty
+	box.prop(prop, "csEndFrame", text = "End Frame")
```

Comment:
the property name automatically gets used as text, no need to duplicate the name here (same with others)
```suggestion
	box.prop(prop, "csEndFrame")
```

Reply:
You are correct and I have changed it for these. However, there are many places throughout the codebase where the name is specified in both places, often with different names.